src/test/java/ch/jalu/collectionbehavior/verification/CollectionMutabilityVerifier.java

```diff
@@ -126,6 +130,71 @@ public static void verifyIsMutableByIteratorAndListIterator(List<String> list) {
         list.clear();
     }
 
+    /**
+     * Verifies that the given Set is mutable (incl. verification that it can be modified via iterator).
+     *
+     * @param emptySet an empty Set instance of the type to test
+     */
+    public static void verifySetIsMutable(Set<String> emptySet) {
+        assertThat(emptySet, empty()); // Validate method contract
+        Set<String> set = emptySet;
+
+        // Set#add, Set#addAll
+        set.add("a");
+        set.add("b");
+        set.addAll(List.of("b", "c", "d", "c", "X", "Y"));
+        assertThat(set, containsInAnyOrder("a", "b", "c", "d", "X", "Y"));
+
+        // Set#remove, Set#removeAll, Set#removeIf
+        set.remove("a"); // b, c, d, X, Y
+        set.removeAll(Set.of("c", "ë")); // b, d, X, Y
+        set.removeIf(elem -> elem.equals("b")); // d, X, Y
+        assertThat(set, containsInAnyOrder("d", "X", "Y"));
+
+        // Set#retainAll
+        set.retainAll(Set.of("d", "X"));
+        assertThat(set, containsInAnyOrder("d", "X"));
+
+        // Set#clear
+        set.clear();
+        assertThat(set, empty());
+    }
+
+    public static void verifyIsMutableByIterator(Set<String> set) {
+        set.add("north");
+        set.add("east");
+        set.add("south");
+        set.add("west");
+
+        Iterator<String> iterator = set.iterator();
+        while (iterator.hasNext()) {
+            iterator.next();
+            iterator.remove();
+        }
+
+        assertThat(set, empty());
+    }
+
+    public static void verifyIsMutableBySequencedSetMethods(SequencedCollection<String> seqColl) {
```

Comment:
list never passed into here. Type more specifically and remove the instanceof check